spec/app/domain/authentication/authn-jwt/functions_spec.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+require 'spec_helper'
+
+RSpec.describe('Authentication::AuthnJwt.path_join') do
+
+  examples = {
+    'with two plain parts': [%w(path host), 'path/host'],
+    'with a trailing slash on left part': [%w(path/ host), 'path/host'],
+    'with a leading slash on right part': [%w(path /host), 'path/host'],
+    'with slashes on both parts': [%w(path/ /host), 'path/host'],
+    'with multiple parts': [%w(a b c d), 'a/b/c/d'],
+    'with leading slash on first part': [%w(/a b c d), '/a/b/c/d'],
+    'with trailing slash on last part': [%w(a b c d/), 'a/b/c/d/'],
+    'with many parts and slashes': [%w(/a/b/c/ /d/e/ f/g), '/a/b/c/d/e/f/g'],
+  }
+
+  examples.each do |description, (input, output)|
```

Comment:
I liked this way of writing short test file. Thank you.
CC : @cyberark/cerberus